Clarify how to contribute

serra opened the pull request:
This is my current understanding of how to contribute to the wiki.
If it makes sense, please consider to merge.

If it doesn't make sense, then we should make it more clear how to contribute, because I should be able to do this.

Cheers!

serra committed:
Update readme to help people get started contributing

By adding this to README.md, we help people who don't have the wiki running yet.

serra committed:
Stop tracking tiddlywiki files that regularly change

But that we don't want to version.
This will avoid accidental commits of these files, as well as contributors
having to manually ignore changes to these files.

jamesgorman2 commented on tiddlers/$__StoryList.tid:
```diff
@@ -1,6 +0,0 @@
-created: 20181111055700759
```
Looks like you've deleted the files while adding the ignores.

serra replied:
Yes, I thought this was the quickest way to go.
I'll ignore them and mark them as assume-unchanged.

serra replied:
And revert the delete.

jamesgorman2 commented on .gitignore:
```diff
@@ -1,2 +1,5 @@
 node_modules
 package-lock.json
+favicon.ico
+$__plugins_felixhayashi_tiddlymap_misc_defaultViewHolder.tid
```
FYI, there are _probably_ more than this, TiddlyWiki has some funny behaviour with timestamps. Not worth doing anything about it yet though.

serra replied:
I'm not sure about the timestamps. It's a bit awkward that were using a version control system *and* use timestamps in the files.

jamesgorman2 replied:
Yes it is. Hopefully we can hide some of that in the interim. When Pete and I talked about this I raised that if you dig into fixing wiki editing too much it starts to dovetail with how the website works, and then into branding and UX and a whole bunch of other stuff that is not as important yet, so a stop-gap seems appropriate for now.

jamesgorman2 reviewed (changes requested):
Thanks Marijn, 

the only change is to revert the deletion of the tiddlers that have been excluded. Otherwise looks good.

FYI, Pete and I were talking about how to script the save button so that it will do all the git things needed to make it easier for less technical coaches to make changes. The plan was to make sure that the script will revert any of these meaningless changes.

serra committed:
Revert "Stop tracking tiddlywiki files that regularly change"

This reverts commit 71b43d8c567363676719de96e74f8089671b46d7.

serra committed:
Add files to gitignore, while keeping them in the repository

Achieved using `git update-index --assume-unchanged <file>`
E.g.: `git update-index --assume-unchanged ./tiddlers/favicon.ico`

This keeps the file in the repository, but removes them from the index so they are no longer tracked.

serra commented:
@jamesgorman2 I think this addresses your issues.
